README.md

```diff
@@ -578,7 +578,7 @@ Using the `validateStatus` config option, you can define HTTP code(s) that shoul
 ```js
 axios.get('/user/12345', {
   validateStatus: function (status) {
-    return status < 500; // Reject only if the status code is greater than or equal to 500
+    return status >= 500; // Reject only if the status code is greater than or equal to 500
```

Comment:
Don't change the code. But you can rewrite the comment as "Resolve only if the status code is less than 500".